sriov_antrea_install.sh

```diff
@@ -0,0 +1,253 @@
+#!/bin/bash -x
+
+export RECLONE=${RECLONE:-true}
+export WORKSPACE=${WORKSPACE:-/tmp/k8s_$$}
+export LOGDIR=$WORKSPACE/logs
+export ARTIFACTS=$WORKSPACE/artifacts
+export TIMEOUT=${TIMEOUT:-300}
+export POLL_INTERVAL=${POLL_INTERVAL:-10}
+
+export ANTREA_CNI_REPO=${ANTREA_CNI_REPO:-https://github.com/vmware-tanzu/antrea.git}
+export ANTREA_CNI_BRANCH=${ANTREA_CNI_BRANCH:-master}
+export ANTREA_CNI_PR=${ANTREA_CNI_PR:-'786'}
+
+export SRIOV_CNI_REPO=${SRIOV_CNI_REPO:-https://github.com/intel/sriov-cni}
```

Comment:
why you need sriov-cni?

Reply:
it was a left over from the copied file. will remove it now.